README.md

```diff
@@ -4,6 +4,42 @@ Plugin for Wordpress sites which have WooCommerce shop plugin installed in them.
 
 Installation guide - https://support.yotpo.com/en/installing-yotpo/woocommerce
 
+# Uploading a new version or changes to https://wordpress.org/plugins/yotpo-social-reviews-for-woocommerce/
+
+1. Download the SVN directory from Wordpress:
+svn checkout https://plugins.svn.wordpress.org/yotpo-social-reviews-for-woocommerce/
+
+ Now this is your working copy of the plugin.
+
+2. Pull the latest changes from github repository: woocommerce-plugin
+
+3. Create a new branch, change the relevant files (changelog, wc_yotpo.php,readme.txt) 
+
+4. Copy the code from  'woocommerce-yotpo' folder to to the working copy under 'trunk' folder. 
+
+5. Verify that the following files are updated: 
+
+        Changelog
+        wc_yotpo.php: update the version in the header
+        readme.txt:  the relevant fields: Stable tag, Tested up to, Requires
+        readme.txt: the changelog section.
+
+6. Add all changes and new files
+svn add * --force
+
+
+7. run 'svn diff' to verify the code changes before you upload the code
+6. Push the changes:
+svn ci -m 'Version 1.1.8' --username Yotpo --password yotpo2010

```

Comment:
I know it's part of the original documentation, but maybe lets change the `1.1.8` part to something like `<<new version number>>` or something? just so nobody accidentally pushes with a wrong commit message (:

Reply:
how does this look ?
svn ci -m 'Version `new version number`' --username Yotpo --password yotpo2010
this symbols <<>> are being treated weirdly in the readme

Reply:
no problem, so long as it's not a fixed number I think it'll do the trick